app/jsx/components/ScholWorksComp.jsx

```diff
@@ -9,6 +10,18 @@ if (!(typeof document === "undefined")) {
   const dotdotdot = require('jquery.dotdotdot')
 }
 
+class DotAuthorUl extends React.Component {
```

Comment:
Is this the 'et al' link that shows up on the search page when a searched-for item has lots of authors? That one isn't currently working for me. It shows up as "et al" but just refreshes the page when I click it. Not a merge-blocker, just fyi.